backend/wmg/pipeline/summary_cubes/cell_count.py

```diff
@@ -63,6 +65,13 @@ def create_cell_count_cube(corpus_path: str):
     """
     obs = extract(corpus_path)
     df = transform(obs)
+
+    cell_types = list(df["cell_type_ontology_term_id"])
+    n_cells = np.array(list(df["n_cells"]))
+    (n_cells_agg,) = aggregate_across_cell_type_descendants(cell_types, [n_cells])
```

Comment:
nit: since this method is sometimes called with an `arrays_to_sum` of size one, you could make this a var args, `*arrays_to_sum`, avoiding having to wrap the to-be-aggregated lists in an explicit list.